terraform/database_dev/providers.tf

```diff
@@ -0,0 +1,30 @@
+terraform {
+  required_version = "~> 1.0"
+  required_providers {
+    cloudfoundry = {
+      source  = "cloudfoundry-community/cloudfoundry"
+      version = "0.12.6"
+    }
+    aws = {
+      version = "~> 3.11.0"
+    }
+  }
+  # backend "s3" {
```

Comment:
[suggestion] i don't think s3 is necessary! we could axe this. (we should anyways if it's commented out)

Reply:
I left it in because I wanted to talk about it, so yay, we're talking about it!

I don't know if it's necessary or not.  From my understanding terraform needs to store its state for each environment somewhere it can refer to on subsequent runs. Can GitHub Actions accomplish that?  If not, I think we need to have an s3 bucket that's holding the state.  Since I was just developing locally for a sandbox, I was writing the state locally, but I didn't want to totally take out references to remote state storage, either, so we could figure out what we want to do about it!